en/integrations/event-management/sc-bsm.md

```diff
@@ -3,124 +3,255 @@ id: sc-hp-bsm
-* A **`root` access in command-line interface on Centreon central server** is required as well.
+Install **Epel** repository.
 
-## Support
+```shell
+yum -y install epel-release
+```
+
+Install dependencies.
+
+```shell
+yum install luarocks make gcc lua-curl lua-devel
+```
+
+<!-- CentOS 8 -->
+
+Install dnf plugins package.
+
+```shell
+dnf -y install dnf-plugins-core
+```
+
+Install **Powertools** repository.
+
+```shell
+dnf config-manager --set-enabled powertools
+```
+
+Install **Epel** repository.
+
+```shell
+dnf -y install epel-release
+```
 
-If you need help with this integration, depending on how you are using Centreon, you can:
+Install dependencies.
 
-* **Commercial Edition customers**: please contact the [Centreon Support team](mailto:support@centreon.com).
-* **Open Source Edition users** or **Centreon IT-100 users** (free versions): please reach our [Community Slack](https://centreon.github.io) where our users and staff will try to help you.
+```shell
+dnf install make gcc libcurl-devel lua-devel luarocks
+```
 
-## Integration Walkthrough
+<!-- RedHat 8 -->
 
-### Installation 
+Install dnf plugins package.
 
-Login as `root` on the Centreon central server using your favorite SSH client.
+```shell
+dnf -y install dnf-plugins-core
+```
 
-In case your Centreon central server must use a proxy server to reach the Internet, you will have to export the `https_proxy` environment variable and configure `yum` to be able to install everything.
+Install **Epel** repository.
 
-```bash
-export https_proxy=http://my.proxy.server:3128
-echo "proxy=http://my.proxy.server:3128" >> /etc/yum.conf
+```shell
+dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
 ```
 
-Now that your Centreon central server is able to reach the Internet, you can run:
+Enable **Codeready** repository.
 
-```bash
-yum install -y lua-curl epel-release
-yum install -y luarocks
+```shell
+subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
+```
+
+Install dependencies.
+
+```shell
+dnf install make gcc libcurl-devel lua-devel luarocks
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+### Lua modules
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--CentOS/Redhat 7-->
+
+Install **luaxml**.
+
+```shell
 luarocks install luaxml
 ```
 
-This package is necessary for the script to run. Now let's download the script:
+Install Centreon lua modules.
+
+```shell
+luarocks install centreon-stream-connectors-lib
+```
+
+<!-- CentOS/Redhat 8-->
 
-```bash
-wget -O /usr/share/centreon-broker/lua/bsm_connector.lua https://raw.githubusercontent.com/centreon/centreon-stream-connector-scripts/master/centreon-certified/bsm/bsm_connector-apiv1.lua
-chmod 644 /usr/share/centreon-broker/lua/bsm_connector.lua
+Install **lua-curl**.
+
+```shell
+luarocks install Lua-cURL
 ```
 
-The BSM StreamConnnector is now installed on your Centreon central server!
+Install **luaxml**.
 
-### Broker configuration
+```shell
+luarocks install luaxml
+```
```

Comment:
this dependency is made using C, does it works on el7 and el8 ? (should be, because i've tested on el8 and it worked and obviously the stream connector has been made using el7 when first created). It's just a comment to about testing things and the issues we can have we dependencies

Reply:
This install is OK.
I'll have to continue my test with the stream connector code